src/patterns/validation/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+title: Recover from errors (validation)
+description: Check the answers users give to make sure they’re valid - and if there’s an error, tell them what's wrong and how to fix it
+section: Patterns
+theme: Help users to…
+aliases: 
+backlog_issue_id: 
+layout: layout-pane.njk
+---
+
+Check the information the user gives you to make sure it’s valid. If there's a validation error, tell the user what's gone wrong and how to fix it. Turn off HTML5 validation.
+
+## When to use this pattern
+
+Validation should be a last resort. Minimise your chances of needing to show an error message by:
+
```

Comment:
Could add something about reasonably normalising input, for example by:
* Stripping leading and trailing whitespace
* Converting / stripping characters you understand but don't need - eg brackets in telephone numbers.
* Converting case on things like postcode and reference numbers.

Reply:
Normalising might be something to cover in other guidance - for now we just want to focus on validation. Another related area we might get into later is sanitation - making input safe to store and display, but yes we wanted to keep the scope just on validation here.